torch/nn/parallel/distributed.py

```diff
@@ -478,6 +478,21 @@ class DistributedDataParallel(Module, Joinable):
                       gradients. If hitting such errors, please fix it by
                       referring to the :meth:`~torch.optim.Optimizer.zero_grad`
                       function in ``torch/optim/optimizer.py`` as a solution.
+        static_graph (bool): When set to ``True``, DDP knows the trained graph is
```

Comment:
It might be helpful for users to understand how to figure out if their graph is static or not (is there an easy check to do this)? The main reason is that sometimes model authoring code is very separated from model training code, so it might not always be completely straightforward.

Reply:
that is a good point! let me add a github issue about this

Reply:
added an example about how to check this in this PR now